### PR TITLE
Fix Typos and Update Comments

### PR DESCRIPTION
**Pull request by RoseSecurity**
## What and Why

- Fixes typos and updates Fleet gen successful response

### CODE REVIEW

File: docs/guides/elasticstack-and-cloud.md [hunk 1]
```diff
@@ -176,4 +176,4 @@ resource "elasticstack_elasticsearch_security_user" "monitoring_user" {
 }
 ```
 
-Note that resources can be targed to certain deployments using the `provider` attribute. 
+Note that resources can be target to certain deployments using the `provider` attribute.
```
Comment (tobio) [inline docs/guides/elasticstack-and-cloud.md]:
```suggestion
Note that resources can be targeted to certain deployments using the `provider` attribute. 
```

File: docs/resources/elasticsearch_transform.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ resource "elasticstack_elasticsearch_transform" "transform_with_pivot" {
 - `defer_validation` (Boolean) When true, deferrable validations are not run upon creation, but rather when the transform is started. This behavior may be desired if the source index does not exist until after the transform is created. Default is `false`
 - `description` (String) Free text description of the transform.
 - `docs_per_second` (Number) Specifies a limit on the number of input documents per second. Default (unset) value disables throttling.
-- `enabled` (Boolean) Controls wether the transform should be started or stopped. Default is `false` (stopped).
+- `enabled` (Boolean) Controls weather the transform should be started or stopped. Default is `false` (stopped).
```
Comment (tobio) [inline docs/resources/elasticsearch_transform.md]:
```suggestion
- `enabled` (Boolean) Controls whether the transform should be started or stopped. Default is `false` (stopped).
```

FWIW I suspect weather is probably just as likely to be correct as wether. In this case neither are the right word.

File: generated/fleet/fleet.gen.go
Comment (tobio) [inline generated/fleet/fleet.gen.go:1]: What's actually returned by the API. Is this a spec issue or does the spec correctly represent a typo in the API?

File: internal/elasticsearch/transform/transform.go [hunk 3]
```diff
@@ -274,7 +274,7 @@ func ResourceTransform() *schema.Resource {
 		},
 		"enabled": {
 			Type:        schema.TypeBool,
-			Description: "Controls wether the transform should be started or stopped. Default is `false` (stopped).",
+			Description: "Controls weather the transform should be started or stopped. Default is `false` (stopped).",
```
Comment (tobio) [inline internal/elasticsearch/transform/transform.go]:
```suggestion
			Description: "Controls whether the transform should be started or stopped. Default is `false` (stopped).",
```

File: templates/guides/elasticstack-and-cloud.md.tmpl [hunk 4]
```diff
@@ -33,4 +33,4 @@ Now we can add resources to these deployments as follows:
 
 {{ tffile "examples/cloud/stack_resources.tf" }}
 
-Note that resources can be targed to certain deployments using the `provider` attribute. 
+Note that resources can be target to certain deployments using the `provider` attribute.
```
Comment (tobio) [inline templates/guides/elasticstack-and-cloud.md.tmpl]:
```suggestion
Note that resources can be targeted to certain deployments using the `provider` attribute. 
```

File: generated/fleet/fleet.gen.go
Comment (RoseSecurity) [reply generated/fleet/fleet.gen.go:1]: I just assumed this was a typo since the other options all return `success`

**Pull request closed without merging**